src/taskpane/server.js

```diff
@@ -72,31 +98,41 @@ var API_PROJECT_BASE = '/services/v6_0/RestService.svc/projects/',
   STATUS_MESSAGE_GOOGLE = {
     1: "All done! To send more data over, clear the sheet first.",
     2: "Sorry, but there were some problems (see the cells marked in red). Check any notes on the relevant ID field for explanations.",
-    3: "We're really sorry, but we couldn't send anything to SpiraPlan - please check notes on the ID fields for more information.",
+    3: "We're really sorry, but there was a problem while sending data to Spira. Some records may had be exported to Spira. Please check it on the application.",
     4: "You are not on the correct worksheet. Please go to the sheet that matches the one listed on the Spira taskpane / the selection you made in the sidebar.",
     5: "Some/all of the rows already exist in SpiraPlan. These rows have not been re-added."
   },
   STATUS_MESSAGE_EXCEL = {
     1: "All done! To send more data over, clear the sheet first.",
     2: "Sorry, but there were some problems (see the cells marked in red). Check any notes on the relevant ID field for explanations.",
-    3: "We're really sorry, but we couldn't send anything to SpiraPlan - please check notes on the ID fields for more information.",
+    3: "We're really sorry, but there was a problem while sending data to Spira. Some records may had be exported to Spira. Please check it on the application.",
     4: "You are not on the correct worksheet. Please go to the sheet that matches the one listed on the Spira taskpane / the selection you made in the sidebar.",
     5: "Some/all of the rows already exist in SpiraPlan. These rows have not been re-added.",
     6: "Data sent! Since you are using the advanced mode, any errors related to comments and/or associations will be marked as red in the spreadsheet. To send more data over, clear the sheet first."
   },
   CUSTOM_PROP_TYPE_ENUM = {
+    101: "StringValue",
+    102: "IntegerValue",
+    103: "DecimalValue",
+    104: "BooleanValue",
+    105: "DateTimeValue",
+    106: "IntegerValue",
+    107: "IntegerListValue",
+    108: "IntegerValue",

```

Comment:
Sure, I had to add a different ID for internal handling, since the addition of new custom types, to avoid mismatches. I added a comment to explain it.